process/project_proposals.adoc

```diff
@@ -0,0 +1,26 @@
+*CNCF Project Proposal Process v1.0*
+
+ . *Introduction*. This governance policy sets forth the proposal process for projects to be accepted into the Cloud Native Computing Foundation (“CNCF”). The process is the same for both existing projects which seek to move into the CNCF, and new projects to be formed within the CNCF.
+ . *Project Proposal Requirements*. Projects must be proposed via email to the +++<u>+++cncf-proposals@lists.cncf.io+++</u>+++ mailing list, with the subject “[PROPOSAL] project name”. Project proposals submitted to the CNCF (see https://docs.google.com/document/d/1AgoEf4Wdshcn2ttlGRVQLciLJe6liV_ZiGocRg9JxJE/edit#[examples]) must provide the following information to the best of your ability:
+
+ .. name of project (must be unique within CNCF)
+ .. project description (what it does, why it is valuable, origin and history)
+ .. statement on alignment with CNCF charter mission
+ .. sponsor from TOC (sponsor helps mentor projects)
+ .. license (charter dictates ASLv2 by default)
+ .. source control (GitHub by default)
+ .. external dependencies (including licenses)
+ .. initial committers (how long working on project)
+ .. infrastructure requests (CI / CNCF Cluster)
+ .. mailing lists (CNCF by default)
+ .. issue tracker (GitHub by default)
+ .. website (current version will move to project.cncf.io)
```

Comment:
if the project currently has a website, where does it live (e.g., http://kubernetes.io/); we've had some initial discussions that when projects move to the CNCF, their website will have to be branded under something like http://kubernetes.cncf.io (adding a simple CNAME alias to your gh-pages repo)